Sources/xcodeproj/Objects/Files/PBXContainerItemProxy.swift

```diff
@@ -37,25 +51,64 @@ public final class PBXContainerItemProxy: PBXObject {
         }
     }
 
+    /// The value of remoteGlobalIDString property in pbxproj. Use this accessor if the object is in remote .xcodeproj.
+    public var remoteGlobalIDString: String? {
+        get {
+            return remoteGlobalIDReference?.value
+        }
+        set {
+            remoteGlobalIDReference = newValue.map { PBXObjectReference($0) }
```

Comment:
I might be wrong because it's been a long time without playing with this project but I'd say the `remoteGlobalIDReference` is not necessary. We can just have `removeGlobalID` of type String.